README.md

```diff
@@ -1,4 +1,5 @@
 West Oakland Air Quality Project (WOAQ) [![TravisCI](https://travis-ci.org/openoakland/woeip.svg?branch=master)](https://travis-ci.org/openoakland/woeip) [![Codecov](https://codecov.io/gh/openoakland/woeip/branch/master/graph/badge.svg)](https://codecov.io/gh/openoakland/woeip)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
```

Comment:
I haven't used `black` before, but it looks pretty simple and there seems to be a [sublime integration](https://github.com/jgirardet/sublack). I'm a bit apprehensive about how unyielding the devs are in this discussion of [single quotes being allowed](https://github.com/psf/black/issues/118), since there isn't really a pythonic principle for preferring one convention over the other. If there *is* a convention we feel strongly about that conflicts with `black` formatting, will we be able to customize it (thinking of django-specific style idiosyncrasies in particular)? 
That point raised, I think having automatic code styling is great in our context. So if everyone on the team feels good about it, SGTM.

Reply:
Yeah, the good or bad thing depending on your perspective is that black tries to encourage you to not modify the convention. So there's no equivalent of a pylintrc file.

However the beauty about black is that you can write using whichever quote style you want and before you commit, black will standardize it. 

Black is new but has been production tested in many python envs especially django ones. In fact when I ran pre-commit on all of our python files, I didn't see it change any django specific settings. Most of the changes come from moving strings to double quotes